zk_toolbox/crates/zk_inception/src/commands/prover/init.rs

```diff
@@ -83,25 +86,19 @@ fn download_setup_key(
         .expect(MSG_PROOF_COMPRESSOR_CONFIG_NOT_FOUND_ERR)
         .clone();
     let url = compressor_config.universal_setup_download_url;
+    let path = std::path::Path::new(path);
+    let parent = path.parent().expect(MSG_SETUP_KEY_PATH_ERROR);
+
+    Cmd::new(cmd!(shell, "wget {url} -P {parent}")).run()?;
+    Cmd::new(cmd!(shell, "mv {parent}/setup_2^24.key {path}")).run()?;
```

Comment:
Why we can't download it to the necessary directory?

Reply:
We download to the correct directory, the second cmd renames the file to the one the user chose.
I couldn't find a way to download to a specific dir + choose the name of the downloaded file at the same time so I do it in two steps